### DISCUSSION TITLE
Missing Gene Families

### DISCUSSION THREAD

**Discussion by Paul-Donat** [hahnlab/CAFE5#166] [Q&A] [answered]
My input file for gene families contained 6,138 gene families. When I look at the Gamma_family_results.txt file it only lists results for 5,873 gene families. Below is the command I used to run cafe.

`cafe5 --infile c1_c2.filt100.tsv --tree c1_c2.tree --cores 40 --n_gamma_cats 15 -p -o results_15`

Why were genes omitted? Any help would be appreciated. Thank you.

**Comment by Kai3005** [comment]
hi paul, i'm confused by the same question as you, and i wonder did you figure it out?

**Comment by benfulton** [comment, marked as answer]
You may see a line in your output like

`Filtering families not present at the root from: 12653 to 10956`

To optimize the process, CAFE doesn't calculate values for families that most likely have a value of 0 at the root. You can override this behavior by specifying the `-z` parameter.
